CHANGELOG.md

```diff
@@ -28,6 +28,12 @@
 
   ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
+- Fix wrapping of long lines of text in summary list
+
+  Thanks to [@MoJ-Longbeard](https://github.com/MoJ-Longbeard) for raising the issue.
```

Comment:
@colinrotherham please add yourself to the thanks here if you want to. 👍

Reply:
Don't worry, I'll still be in `git blame` for all time